lib/identity_cache/cached/attribute.rb

```diff
@@ -0,0 +1,158 @@
+# frozen_string_literal: true
+
+module IdentityCache
+  module Cached
+    class Attribute
+      attr_reader :model, :attribute, :alias_name, :key_fields, :unique
+
+      def initialize(model, attribute, alias_name, key_fields, unique)
+        @model = model
+        @attribute = attribute
```

Comment:
Does this need to be symbolized too?

Reply:
It wasn't before, but it would probably be better to be clear about its type.